### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by PatrykMis**
Replaces https://github.com/ReVanced/revanced-manager-compose/pull/90

Not tested yet, will do it tomorrow.

### CODE REVIEW

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -1,31 +1,31 @@
 [versions]
-ktx = "1.10.1"
-viewmodel-lifecycle = "2.6.1"
+ktx = "1.12.0"
```
Comment (validcube) [inline gradle/libs.versions.toml:2]: Bumping ktx version to 1.12.0 will fail because it was compiled against 34, our application was compiled against 33.

**Review by validcube — CHANGES_REQUESTED** [review]

**Review by validcube — CHANGES_REQUESTED** [review]
Upon compiling I see error related to `androidx.emoji2:emoji2-views-helper:1.4.0` compiled for Android 14, not Android 13.

**Comment by PatrykMis** [comment]
Ok, fixed and now tested.

File: app/build.gradle.kts [hunk 2]
```diff
@@ -4,7 +4,7 @@ plugins {
     alias(libs.plugins.devtools)
     alias(libs.plugins.about.libraries)
     id("kotlin-parcelize")
-    kotlin("plugin.serialization") version "1.9.0"
+    kotlin("plugin.serialization") version "1.9.10"
```
Comment (Axelen123) [inline app/build.gradle.kts:7]: The previous `kotlinx.serialization` bump caused a crash at runtime. Can you get past the auto updates dialog with both options selected without crashing?

Comment (PatrykMis) [reply app/build.gradle.kts:7]: Yes that works, at least for me on Pixel 4 device with Android 13.

**Review by validcube — APPROVED** [review]
🌟 +1 Approved, Tested on Debug variant of ReVanced Manager Compose

**Review by Axelen123 — APPROVED** [review]

**Comment by Axelen123** [comment]
Apologies for the late review

**Pull request merged**